<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graficos</class>
 <widget class="QMainWindow" name="Graficos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QCustomPlot" name="graficoAcX" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graficoAcY" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graficoAcZ" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>571</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
